pkgs/by-name/ta/tacentview/package.nix

```
@@ -0,0 +1,68 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  libGL,
+  ninja,
+  stdenv,
+  tacent,
+  xorg,
+  zenity,
+  ...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tacentview";
+  version = "1.0.46";
+
+  src = fetchFromGitHub {
+    owner = "bluescan";
+    repo = pname;
+    rev = version;
+    hash = "sha256-d4A26p1hmkYEZ+h6kRbHHr4QmAc3PMe3qYdkeKIRGkU=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+  ];
+
+  buildInputs = [
+    libGL
+    tacent
+    xorg.libX11
+    xorg.libxcb
+    zenity
+  ];
+
+  cmakeFlags = [
+    "-DFETCHCONTENT_SOURCE_DIR_TACENT=${tacent.src}"
+    "-DPACKAGE_NIX=True"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D tacentview -t $out/bin
```

```suggestion
    installBin tacentview
```

and put `installShellFiles` in `nativeBuildInputs`.